src/m365/spo/commands/web/web-set.ts

```diff
@@ -169,6 +175,9 @@ class SpoWebSetCommand extends SpoCommand {
     if (typeof args.options.footerEnabled !== 'undefined') {
       payload.FooterEnabled = args.options.footerEnabled;
     }
+    if (typeof args.options.navAudienceTargetingEnabled !== 'undefined') {
+      payload.navAudienceTargetingEnabled = args.options.navAudienceTargetingEnabled;
```

Comment:
The property should start with a capital letter, otherwise it will not be recognized.

Reply:
good catch!